packages/components-react/package.json

```diff
@@ -0,0 +1,84 @@
+{
+  "version": "1.0.0-alpha.340",
+  "author": "Community for NL Design System",
+  "description": "React component library for the Example repository, based on the NL Design System architecture",
+  "license": "EUPL-1.2",
+  "name": "@example/components-react",
+  "keywords": [
+    "nl-design-system"
+  ],
+  "private": true,
+  "publishConfig": {
+    "access": "restricted",
+    "registry": "https://registry.npmjs.org/"
+  },
+  "repository": {
+    "type": "git+ssh",
+    "url": "git@github.com:nl-design-system/example-community.git"
+  },
+  "scripts": {
+    "prebuild": "npm run clean",
+    "build": "npm-run-all build:**",
+    "build:components": "vite build",
+    "clean": "rimraf dist/ pages/",
+    "lint": "tsc --project ./tsconfig.json --noEmit && tsc --noEmit --project ./tsconfig.test.json",
+    "test": "mkdir -p pages && NODE_OPTIONS=--experimental-vm-modules jest --verbose",
+    "watch:test": "NODE_OPTIONS=--experimental-vm-modules jest --verbose --watch"
+  },
+  "main": "./dist/index.cjs.js",
+  "module": "./dist/index.esm.js",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist/"
+  ],
+  "dependencies": {
+    "clsx": "1.2.1",
+    "date-fns": "2.30.0",
+    "lodash.chunk": "4.2.0",
+    "react-sparkle": "2.0.0"
+  },
+  "devDependencies": {
+    "@babel/core": "7.21.5",
+    "@babel/plugin-transform-runtime": "7.21.4",
+    "@babel/preset-env": "7.21.5",
+    "@babel/preset-react": "7.18.6",
+    "@babel/preset-typescript": "7.21.5",
+    "@babel/runtime": "7.21.5",
+    "@rollup/plugin-babel": "6.0.3",
+    "@rollup/plugin-commonjs": "24.1.0",
+    "@rollup/plugin-node-resolve": "15.0.2",
+    "@testing-library/dom": "8.20.0",
+    "@testing-library/jest-dom": "5.16.5",
+    "@testing-library/react": "13.4.0",
+    "@testing-library/user-event": "14.4.3",
+    "@types/date-fns": "2.6.0",
+    "@types/jest": "29.5.1",
+    "@types/lodash.chunk": "4.2.7",
+    "@types/react": "18.2.0",
+    "@types/testing-library__jest-dom": "5.14.5",
+    "@vitejs/plugin-react": "4.0.2",
+    "jest": "29.5.0",
+    "jest-environment-jsdom": "29.5.0",
+    "next": "13.3.4",
+    "npm-run-all": "4.1.5",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "rimraf": "4.4.1",
+    "rollup": "3.26.3",
+    "rollup-plugin-filesize": "10.0.0",
+    "rollup-plugin-node-externals": "6.1.1",
+    "rollup-plugin-node-polyfills": "0.2.1",
+    "rollup-plugin-peer-deps-external": "2.2.4",
+    "rollup-plugin-postcss": "4.0.2",
```

Comment:
The dependency `rollup-plugin-postcss` appears to be unused currently.

Rather than deleting the dependency, I suggest we start using it. Utrecht already does, works well there.

I think it would be nice for the example repository to have support for injecting `.scss` from the `@example/components-css`.

Reply:
Updated rollup build to support .scss